src/Compilers/CSharp/Portable/Parser/DocumentationCommentParser.cs

```diff
@@ -1262,8 +1262,14 @@ private CrefParameterSyntax ParseCrefParameter()
                     break;
             }
 
+            SyntaxToken readOnlyOpt = null;
+            if (CurrentToken.Kind == SyntaxKind.ReadOnlyKeyword && refKindOpt is { Kind: SyntaxKind.RefKeyword })
```

Comment:
>if (CurrentToken.Kind == SyntaxKind.ReadOnlyKeyword && refKindOpt is { Kind: SyntaxKind.RefKeyword }) [](http://example.com/codeflow?start=12&length=101)

Consider dealing with "unexpected" `readonly` the same way we deal with unexpected `unchecked` in ```TryEatCheckedKeyword```.

Reply:
I guess we still need not null ```refKindOpt``` for that.